engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -431,6 +431,24 @@ public void testTypeSignature() throws Exception {
     """);
   }
 
+  @Test
+  @Ignore
```

Comment:
I see. Neither `resolve : ...` nor `id x = x` is supported. I'll work on that.

Reply:
I have the implementation. It works, but a single issue - I need to check if an identifier is an operator or not. The code is here https://github.com/enso-org/enso/blob/3e03e7db3a1237e14bf20dcc207351c45bad81e1/engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java#L1826 - is there any recommended way to find out a `Token` is an operator?

I am getting `Token.Ident` even for `==` with `toString()` value `Ident["", "==", false, 0]`

Reply:
For now recognized manually: https://github.com/enso-org/enso/pull/3611/commits/51efba96174488d678b687fdc9850a77bb139d86